src/main/thrift/Channels.thrift

```diff
@@ -22,6 +23,36 @@ typedef Aroma.long long
 typedef Aroma.timestamp timestamp
 typedef Aroma.uuid uuid
 
+struct IOSDevice
+{
+    /** Device Token may be stored and serialized as a Base64 encoded String. */
+    1: string deviceToken;
+}
+
+/** The key used to store the serialized Message Object. **/
+const string PUSH_NOTIFICATION_KEY_FOR_MESSAGE = 'aroma.message';
```

Comment:
Unclear if this is the best place to put this. Might be better off in a `Notifications.thrift` file.